package.json

```diff
@@ -67,9 +67,9 @@
         "onCommand:makefile.outline.cleanConfigure",
         "onCommand:makefile.outline.preConfigure",
         "onCommand:makefile.resetState",
-        "workspaceContains:makefile",
-        "workspaceContains:Makefile",
-        "workspaceContains:GNUmakefile"
+        "workspaceContains:**/makefile",
```

Comment:
Somehow I missed this. We weren't supposed to activate the extension for makefiles anywhere in the workspace until we could ensure it wouldn't interfere with CMake Tools. VS Code fixed the activation on timeout problem, but do we do the right thing when CMake generates "Unix Makefiles"?